Gems/Terrain/Code/Source/Components/TerrainPhysicsColliderComponent.h

```diff
@@ -111,9 +111,9 @@ namespace Terrain
         void GenerateHeightsInBounds(AZStd::vector<float>& heights) const;
         AZ::Aabb GetRegionClampedToGrid(const AZ::Aabb& region) const;
 
-        void NotifyListenersOfHeightfieldDataChange(const AZ::Aabb* dirtyRegion = nullptr,
-            Physics::HeightfieldProviderNotifications::HeightfieldChangeMask heightfieldChangeMask =
-                Physics::HeightfieldProviderNotifications::HeightfieldChangeMask::Unspecified);
+        void NotifyListenersOfHeightfieldDataChange(
+            Physics::HeightfieldProviderNotifications::HeightfieldChangeMask heightfieldChangeMask,
+            const AZ::Aabb* dirtyRegion = nullptr);
```

Comment:
Suggestion: I know this wasn't a part of this change, but consider making "AZ::Aabb*" a reference instead. It's already possible to check Aabb::IsValid(), so I'm not sure if there's any value in allowing null pointers to be passed around.

Reply:
Good idea! 
Merged MaterialData & SurfaceData together + passing AZ::Aabb by const ref looks better